src/pages/Storefront/store.js

```diff
@@ -1,18 +1,29 @@
+import { ChakraProvider } from '@chakra-ui/react';
 import React, { useEffect } from 'react';
+import { Navbar, BestSellers, ItemListings, Footer } from '../../components/Storefront';
 import { useShopify } from '../../hooks/useShopify';
+import storeTheme from '../../themes/store';
 
 const Store = () => {
-  const { createShop, createCheckout, fetchProducts } = useShopify();
+  const { createShop, createCheckout, fetchProducts, fetchCollections } = useShopify();
 
   useEffect(() => {
     // Component on mount (i.e. app init): Try to fetch user data (Apollo client internally uses a cookie)
     createShop();
     fetchProducts();
+    fetchCollections();
     createCheckout();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  return <div>Storefront</div>;
+  return (
+    <ChakraProvider theme={storeTheme}>
+      <Navbar />
+      <BestSellers />
+      <ItemListings />
+      <Footer />
```

Comment:
`Navbar`, `ItemListings`, and `Footer` are all layout placeholder components to be implemented

Reply:
thanks for putting this in!